src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -97,14 +97,25 @@ const Comments = ({
       });
   };
 
+  const resolveComment = (commentId: string, resolved: boolean) => {
+    const index = comments.findIndex(comment => comment.id === commentId);
+    releaseContentCommentService
+      .updateContentSectionComment(comments[index], resolved)
```

Comment:
Instead of mixing this in with the existing `updateContentSectionComment` method, why not spin off a separate `setCommentResolution` service method?  Then `updateContentSectionComment` doesn't need the `resolved: boolean | null` property.

Looking at the code paths, updating any other details on the comment and setting the Resolved value are mutually exclusive anyway.

Reply:
As discussed on standup, I'm leaving this as it is for now until we decide whether to split the resolved functionality into a separate endpoint.